### PR TITLE
fix(@clayui/css): LPD-39960 Fix padding between text and icon on table head

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPD-39960

image

### CODE REVIEW

File: packages/clay-css/src/scss/variables/_tables.scss [hunk 1]
```diff
@@ -325,7 +325,7 @@ $c-table: map-deep-merge(
 		autofit-col: (
 			justify-content: center,
 			padding-left: nth($table-cell-padding, 2),
```
Comment (pat270) [inline packages/clay-css/src/scss/variables/_tables.scss:327]: We should change this one to 0.125rem as well.

**Review by pat270 — CHANGES_REQUESTED** [review]
@ilzamcmed We also need to update it in cadmin.

File: packages/clay-css/src/scss/variables/_tables.scss (hunk 1 above)
Comment (ilzamcmed) [reply packages/clay-css/src/scss/variables/_tables.scss]: Hey @pat270 , I was discussing it with @ethib137 and found best to add `0.5rem` on each padding.

Comment (ilzamcmed) [reply packages/clay-css/src/scss/variables/_tables.scss]: I will add this to cadmin. Thanks!

File: packages/clay-css/src/scss/variables/_tables.scss [hunk 2]
```diff
@@ -324,8 +324,8 @@ $c-table: map-deep-merge(
 		),
 		autofit-col: (
 			justify-content: center,
-			padding-left: nth($table-cell-padding, 2),
-			padding-right: nth($table-cell-padding, 2),
+			padding-left: nth($table-cell-padding, 1),
```
Comment (ilzamcmed) [inline packages/clay-css/src/scss/variables/_tables.scss:327]: @ethib137 , this will add `0.5 rem`, so I removed the `calc()` function

Comment (pat270) [reply packages/clay-css/src/scss/variables/_tables.scss:327]: @ilzamcmed this is referencing the top and bottom padding of a table cell. I don't think we should do this.

Comment (pat270) [reply packages/clay-css/src/scss/variables/_tables.scss:327]: I also found use cases in Portal that are using utilities to put 0.25rem padding left and right on autofit-col's in tables. @ethib137 would 0.25rem be acceptable? I think 4px/8px spacing is what Lexicon generally uses

Comment (ilzamcmed) [reply packages/clay-css/src/scss/variables/_tables.scss:327]: Thank you, @pat270! Evan clarified the function of `nth`, as I had misunderstood it. I will wait for Evan's feedback regarding your suggestion and will send the necessary fix afterward.

Comment (ilzamcmed) [reply packages/clay-css/src/scss/variables/_tables.scss:327]: Hey @marcoscv, what are your thoughts on Patrick's comment above?

Comment (marcoscv-work) [reply packages/clay-css/src/scss/variables/_tables.scss]: Hey guys, sorry for the delay, marcoscv is my personal user and I don’t get notifications.
From my side any way to fix it is ok, you guys are the experts! 🎉

**Review by pat270 — APPROVED** [review]

**Comment by ilzamcmed** [comment]
Thanks @pat270 ! :bowing_woman:

**Pull request merged**